cfgov/v1/documents.py

```diff
@@ -116,9 +154,16 @@ def filter_archived(self, search):
         return search.filter("terms", is_archived=self.archived)
 
     def search_title(self, search):
-        return search.query(
-            "match", title={"query": self.title, "operator": "AND"}
-        )
+        if flag_enabled('EXPAND_FILTERABLE_LIST_SEARCH'):
+            query = MultiMatch(
+                query=self.title,
```

Comment:
I'm not getting expected results from the title search with the flag enabled.

Example: I would expect, with the 'EXPAND_FILTERABLE_LIST_SEARCH' flag enabled, that content from the `preview_description` field would be indexed, but a search for "pagkalat," which is in the search description field of live Blog page 13848, doesn't show up in search. 

I wonder if this line should be 


```suggestion
                query=search,
```

Reply:
Ah, I see self.title is the search title param. Still, I'm not getting why the search description term isn't found.

Also, is it intended behavior that the "item name" search only matches an exact phrase in a title?

Reply:
Do I have this right? SublandingFilterablePage (/about-us/blog/) doesn't hit Elasticsearch? The FilterableListMixin just gets querysets?

Reply:
So you will need two flags on - `EXPAND_FILTERABLE_LIST_SEARCH` and `ELASTICSEARCH_FILTERABLE_LISTS` both enabled. 

We also don't explicitly index the search_description but rather the preview_description (I think they are identical values in most cases though)

Reply:
All questions cleared up after enabling the 'ELASTICSEARCH_FILTERABLE_LISTS' flag as well as 'EXPAND_FILTERABLE_LIST_SEARCH'.

I'm now seeing expected results from the search_description field, including my test case, "pagkalat," and multi-term searches.